### ISSUE TITLE
Meteor limit is too low

### ISSUE THREAD

**Issue by Playtester** [rathena/rathena#2217]
**rAthena Hash**: 20908ee

**Client Date**: 2015

**Server Mode**: Both

**Description of Issue**: When I reimplemented Meteor Storm and Catnip Meteor I changed it from using "timerskill" to using unit groups for the timer. In the previous implementation, I already calculated that on Aegis you can have a maximum of 50 Meteors active at the same time (in renewal, where minimum delay is 600ms) and set MAX_SKILLTIMERSKILL to 50. However, now that it uses unit groups, actually a different max applies:

#define MAX_SKILLUNITGROUP 25 /// Maximum skill unit group (for same skill each source)

The max is only 25, causing up to half of the meteors to disappear before dealing damage when spaming the spell at 600ms intervals.

In short, we need to increase the max to 50 here as well:

#define MAX_SKILLUNITGROUP 50 /// Maximum skill unit group (for same skill each source)

**Comment by Playtester** [comment]
I'm not fully sure, but could be that this needs to be increased as well:

#define MAX_SKILLUNITGROUPTICKSET 25
